### DISCUSSION TITLE
How to only see a thread once?

### DISCUSSION THREAD

**Discussion by Johneltonsbrain** [protz/thunderbird-conversations#1627] [General]
If a conversation has 5 Emails, I would love to only see one of them  in my Inbox so that there are less Emails. I'm sure its possible to set it up like that I just dont know how!

**Comment by Johneltonsbrain** [comment]
View-threaded. Omg this extension is amazing.

**Comment by Bappamaiti** [comment]
![image](https://user-images.githubusercontent.com/81735229/128592116-260e9b76-3c1f-42c2-acea-f0789170e0e6.png)

Go to options of this add on and turn on the expand for all messages. Hope this is what you wanted.
